src/services/breakpoints.js

```diff
@@ -1,26 +1,36 @@
-let breakpoints = {};
+import sassVariables from 'homeday-blocks/src/styles/variablesForJS.scss';
 
-export const matchMediaAvailable = (
-  typeof window !== 'undefined'
-  && typeof window.matchMedia === 'function'
-);
+let breakpoints = {
+  mobile: `(max-width: ${sassVariables.breakMobile - 1}px)`,
```

Comment:
So, would we like to remove this one?

Reply:
Yes. And this PR is `on hold`.
There is a ticket to update the breakpoints:
https://homeday.atlassian.net/browse/FET-156

I'll tackle it next sprint.